### PR TITLE
Import script to convert macomber text file & incipits into CSV

### PR DESCRIPTION
**Pull request by rlskoeser**

### CODE REVIEW

File: scripts/macomber_to_csv.py [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+        'GVE': 'G-Vatican (BAV)',
+        'HBS': 'H-Berlin (BS)',
+        'LUE': 'L-Uppsala (UU)',
+        'SALE': 'S-Rome (ANL)',
+        'SBLE': 'S-London (SBL)',
+        'SGE': 'S-Paris (SBN)',
+        'SWE': 'M-SWE (SWE)',
+        'VLVE': 'VL-Vatican (BAV)',
+        'WBLE': 'W-London (BM)',
+        'PEth': 'PEth (PEth)',    # should this be PUL ?
```
Comment (thatbudakguy) [inline scripts/macomber_to_csv.py]: makese sense to me for it to be PUL. I feel like we've talked about this before and agreed that it would be PUL?

Comment (rlskoeser) [reply scripts/macomber_to_csv.py]: I think so, too - this is one of my open questions for Wendy.

**Review by thatbudakguy — COMMENTED** [review]
a few general comments, but this seems like it's working as intended.

- docblocks on methods would be nice to have? the inline comments are great but a little summary of each method is even better, imo. i know this is theoretically a run-once thing so not that important.
- could the script output its results back into the `data/` folder, since we have that now? or alternatively maybe we want to store the "processed" data somewhere separate from the "raw"; not sure. should wherever it dumps output should be `.gitignore`'d too?
- I see a lot of canonical stories with quotes in them. could we either quote all of them or none of them?

a few other oddities I randomly noticed looking at the canonical stories (probably these are the result of errors that are already listed in the output, but just for fun):
- 121 has some weird OCR - `ti!m®momm`
- 215 and 235 are an empty row
- 499 has a typo - `concering`
- 595 has a `None` at the end?
- 602 has a typo - `feven`
- 615 has page info in it
- 636 has a typo (i think????) - `rnrommnrprom`
- 642 has what looks like an OCR typo - `1etos`

**Comment by rlskoeser** [comment]
- Agreed, doc blocks for methods are a good idea
- I wondered about script output; what if it goes into a date-stamped folder? e.g. 2020-01-28_output or similar? I wondered about making another argument to the script but I'm lazy and didn't want to have to type another option
- I think the quotes in the canonical stories is just a CSV output formatting convention to handle content with commas; when I import them into google sheets there are no quotes

Thanks for the list of oddities; I will check on the empty rows and Nones, those seem possibly concerning.

There are lots of OCR errors still to be corrected; my inclination is to ignore them unless they cause problems in the import script since they can be fixed in google sheets after import.

**Comment by thatbudakguy** [comment]
ah, I wondered if that was the case for the quotes - thanks for checking.

I'm conflicted about whether or not to store the output in git at all...maybe we could have it output to a different directory that is ignored, but then run once and timestamp it and add that to the `data/` folder?

File: scripts/macomber_to_csv.py (hunk 1 above)
Comment (thatbudakguy) [reply scripts/macomber_to_csv.py]: Wendy [confirmed via slack](https://cdhprinceton.slack.com/archives/CJ2V8M0B0/p1580227112020500) this should be PUL:
> Re PEth, yes, excellent catch! I should be PUL.

**Review by thatbudakguy — APPROVED** [review]

**Pull request merged**